web/modules/custom/joinup_core/src/JoinupRelationManagerInterface.php

```diff
@@ -128,7 +128,7 @@ public function getUserMembershipsByRole(AccountInterface $user, string $role, a
    * @return \Drupal\rdf_entity\Entity\Rdf[]
    *   An array of collections.
    */
-  public function getCollectionsWhereSoleOwner(AccountInterface $user): array;
+  public function getGroupsWhereSoleOwner(AccountInterface $user): array;
```

Comment:
Now that we have the `joinup_group` module, I think is worth to move this method there, in a new service, something like `JoinupGroupManager`, or any other better naming.

Reply:
I would really prefer not to do that now. I attempted to do it but I got into a series of dependencies that I don't grasp right now (I haven't checked the other ticket in total). For example, the method is using the `::getUserMembershipsByRole` which is another method in the relation manager and then this method is simply fetching memberships by roles which brings the questions at which point should this be moved as well or the `JoinupRelationManager` should be a dependency of the `joinup_group` module and then what happens with the rest of the methods etc etc.... So, can we skip that and leave it to be dealt with entirely in the other ticket?

Reply:
After looking at the rest of the comment, I have moved also the relevant method to the joinup_group service. I will move as much code as possible because it will help relax the work on the follow up ticket.